tests/schema/models.py

```diff
@@ -124,6 +124,13 @@ class Meta:
         db_table = "schema_note"
 
 
+class NoteInfoIndexed(models.Model):
```

Comment:
`NoteInfoIndexed` needs to be created conditionally. Now this is an inner model.